.golangci.yml

```diff
@@ -42,6 +42,9 @@ run:
   timeout: 10m
   modules-download-mode: vendor
   skip-dirs-use-default: true
+  skip-files:
+    - '.*\.gen\.go$' 
+
```

Comment:
I'm not 100% on why we would skip these files in the linter. Is there a reason the generated code files can't be made to pass?

Reply:
I added it since the name of the generated directives map and matchFunc may be long and can't pass funlen lint.

Reply:
I think you may be able to disable specific rules through the `golangci-lint` config if the file matches a pattern. You could match `.gen.go` files and disable linters that are hard to accommodate vs disable all linters.

Reply:
Applied: only disable funlen for `.gen.go` files